### PR TITLE
COOPR-551: Tenant and Provisioner CLI commands integration tests

### PR DESCRIPTION
**Pull request by ghost**
Please review this PR after merging https://github.com/caskdata/coopr/pull/714 and https://github.com/caskdata/coopr/pull/716

### CODE REVIEW

File: coopr-integration-testing/src/test/java/co/cask/coopr/test/cli/TenantCommndsTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  @BeforeClass
+  public static void beforeClass()
+    throws URISyntaxException, NoSuchFieldException, IllegalAccessException, IOException {
+    createCli(SUPERADMIN_ACCOUNT);
+  }
+
+  @Test
+  public void testListTenants() throws InvalidCommandException, UnsupportedEncodingException {
+    execute(Constants.LIST_TENANTS_COMMAND);
+    Set<TenantSpecification> result = getSetFromOutput(new TypeToken<List<TenantSpecification>>() {}.getType());
```
Comment (albertshau) [inline coopr-integration-testing/src/test/java/co/cask/coopr/test/cli/TenantCommndsTest.java]: this should pass in a set type right?

**Comment by albertshau** [comment]
small question, otherwise lgtm

File: coopr-integration-testing/src/test/java/co/cask/coopr/test/cli/TenantCommndsTest.java (hunk 1 above)
Comment (ghost) [inline coopr-integration-testing/src/test/java/co/cask/coopr/test/cli/TenantCommndsTest.java]: yes

**Comment by albertshau** [comment]
lgtm pending travis

**Pull request merged**